### PR TITLE
Replacing the Depricated API

### PR DESCRIPTION
**Pull request by amitsrivastava78**
tf.initialize_all_variables() is not supported replace with tf.global_variables_initializer

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 16 days with no activity and the `awaiting review` label has been applied.

**Comment by amitsrivastava78** [comment]
@aselle , can you pls review this PR

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle, @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/feature_column/feature_column_v2.py [hunk 1]
```diff
@@ -2702,7 +2702,8 @@ def _transform_input_tensor(self, input_tensor):
           'SparseTensor is not supported. key: {}'.format(self.key))
     if self.normalizer_fn is not None:
       input_tensor = self.normalizer_fn(input_tensor)
-    return math_ops.to_float(input_tensor)
+    #return math_ops.to_float(input_tensor)
```
Comment (rohan100jain) [inline tensorflow/python/feature_column/feature_column_v2.py]: Can you remove these comment lines?

**Review by rohan100jain — CHANGES_REQUESTED** [review]
Overall changes look good. Just please remove the commented lines out.

**Comment by rthadur** [comment]
@amitsrivastava78 gentle  ping to check reviewer comments

File: tensorflow/python/feature_column/feature_column_v2.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/python/feature_column/feature_column_v2.py]: Thanks @rohan100jain , i have updated the code as per comments, kindly check.

**Comment by amitsrivastava78** [comment]
@rthadur , thanks for the reminder, i just missed these comments,

**Review by rohan100jain — DISMISSED** [review]

**Comment by rthadur** [comment]
@amitsrivastava78 please resolve conflicts

**Comment by amitsrivastava78** [comment]
@rthadur , this PR is quite old and some one has already merged these changes, so i am closing the PR.

Regards
Amit

**Pull request closed without merging**